src/parser/core/Combatant.ts

```diff
@@ -276,8 +276,12 @@ class Combatant extends Entity {
     return Boolean(this.conduitsByConduitID[spellId]);
   }
 
-  conduitRankBySpellID(spellId: number) {
-    return this.conduitsByConduitID[spellId] && this.conduitsByConduitID[spellId].rank;
+  conduitRankBySpellID(spellId: number): number {
+    if (!(spellId in this.conduitRankBySpellID)) {
```

Comment:
this.conduitRankBySpellID doesn't seem right, did you mean conduitsByConduitId

Reply:
🤫